config/cloud.cfg.tmpl

```diff
@@ -149,6 +149,8 @@ system_info:
    # This will affect which distro class gets used
 {% if variant in ["amazon", "arch", "centos", "debian", "fedora", "freebsd", "netbsd", "openbsd", "rhel", "suse", "ubuntu"] %}
    distro: {{ variant }}
+{% elif variant in ["dragonfly"] %}
+   distro: freebsd
```

Comment:
This means we're overriding the distro here.

Reply:
Yes, but I've changed that in my last iteration.